autonomy/deploy/generators/docker_compose/base.py

```diff
@@ -153,31 +146,29 @@ def generate_config_tendermint(
 
         if "Unable to find image" in cast(IO[str], process.stderr).read():
             raise RuntimeError(
-                f"Cannot find {TENDERMINT_IMAGE_NAME}:{image_version}, Please build images first."
+                f"Cannot find {TENDERMINT_IMAGE_NAME}:{TENDERMINT_IMAGE_VERSION}, Please build images first."
             )
         return self
 
     def generate(
         self,
-        image_versions: Dict[str, str],
     ) -> "DockerComposeGenerator":
         """Generate the new configuration."""
 
         agent_vars = self.service_spec.generate_agents()
-        image_name = self.service_spec.service.agent.name
-
-        if self.dev_mode:
-            image_versions["agent"] = "dev"
+        runtime_image = OAR_IMAGE.format(
+            agent=self.service_spec.service.agent.name,
+            version=self.service_spec.service.agent.hash,
```

Comment:
So we need to use hashes as tag for agent runtime images since we don't update the versions for agent packages yet.

Reply:
This would be redundant until we introduce versioning for packages, I suggest we do this once we introduce versioning.